cogito/putter.go

```diff
@@ -133,7 +133,7 @@ func (putter *ProdPutter) ProcessInputDir() error {
 	case 1:
 		repoDir := filepath.Join(putter.InputDir, inputDirs.OrderedList()[0])
 		putter.log.Debug("", "inputDirs", inputDirs, "repoDir", repoDir, "msgDir", msgDir)
-		if err := checkGitRepoDir(repoDir, source.Owner, source.Repo); err != nil {
+		if err := checkGitRepoDir(repoDir, putter.ghAPI, source.Owner, source.Repo); err != nil {
```

Comment:
integration tests fail due to this input change.
`putter.ghAPI` is `https://api.github.com`; from this domain name is extracted: `api.github.com` and compared to what is configured in the `.git/config`, which is `github.com`. This doesn't match and the test fails..